cputest/frsp.cpp

```diff
@@ -15,9 +15,11 @@ static void FrspTest()
 		{0x000fffffffffffff, 0x0000000000000000, 0b000}, // largest double subnormal
 		{0x3690000000000000, 0x0000000000000000, 0b000}, // largest number rounded to zero
 		{0x3690000000000001, 0x36a0000000000000, 0b000}, // smallest positive single subnormal
+		{0x380fffffe0000001, 0x0000000000000000, 0b110}, // boundary of architecture-dependent behavior in non-IEEE mode
```

Comment:
Personally I think it would make more sense to test this using the largest single subnormal (and equivalently the smallest single subnormal below), assuming that that does indeed produce the same result.